modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/AddressEndpointFactory.java

```diff
@@ -59,6 +62,8 @@
  */
 public class AddressEndpointFactory extends DefaultEndpointFactory {
 
+    private static final String SYSTEM_VARIABLE_PREFIX = "$SYSTEM";
+    private static final Log LOG = LogFactory.getLog(AddressEndpointFactory.class);
```

Comment:
are we using this anywhere in the code?

Reply:
Resolved https://github.com/apache/synapse/commit/048fe2d6c440af65669c13830d486d9fa7a741fa